src/Evaluator.ml

```diff
@@ -15,17 +15,41 @@ let rec free_vars_g term =
   | Fun (_i, _t_of_id, body) -> free_vars_g body
   | App (fe, arge) -> Set.union (free_vars_g fe) (free_vars_g arge)
   | Box e -> free_vars_g e
+  | Let (_i, _bounded_e, body) -> free_vars_g body
```

Comment:
Here, we should also walk over the bound expression (and take the set union).
Because we want the following example
```
dune exec -- mtt eval -e"
letbox x' =
  box (let z = y' in z)
in
letbox y' = box () in
x'
"
```
to fail (it accesses the free modal variable `y'`).